lib/image-inspector-client.rb

```diff
@@ -51,12 +51,7 @@ def fetch_metadata
     def exception_handler
       yield
     rescue RestClient::Exception => e
-      begin
-        json_error_msg = JSON.parse(e.response || '') || {}
-      rescue JSON::ParserError
-        json_error_msg = {}
-      end
-      err_message = json_error_msg['message'] || e.to_s
+      err_message = e.response || e.message || ''
```

Comment:
- Nice :+1: for this direction.
- I think you need ```e.response.body```please check that.
- I'll be good with this but lets talk and see if we have an effective way to test rest-client api - we really need to make sure exceptions work as we expect them